includes/admin/class-wc-rest-stripe-account-controller.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Class WC_REST_Stripe_Account_Controller
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for retrieving Stripe's account data.
+ *
+ * @since 5.6.0
+ */
+class WC_REST_Stripe_Account_Controller extends WC_Stripe_REST_Controller {
+	/**
+	 * Endpoint path.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'wc_stripe/account';
+
+	/**
+	 * Configure REST API routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			[
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => [ $this, 'get_account' ],
+				'permission_callback' => [ $this, 'check_permission' ],
+			]
+		);
+	}
+
+	/**
+	 * Retrieve the Stripe account information.
+	 *
+	 * @return WP_REST_Response
+	 */
+	public function get_account() {
+		return new WP_REST_Response( WC_Stripe_API::retrieve( 'account' ) );
```

Comment:
Can [this](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/feat/account-wp-data-implementation/includes/class-wc-stripe-account.php#L46) be reused here?

Reply:
The `wp.data.dispatch('wc/stripe').refreshAccount()` uses this same method to refresh the account data on the frontend.
If we used `get_cached_account_data`, we would get the same data, without refreshing it.

Reply:
I've been thinking about it and I also think it would be good to use that class - I refactored things a bit in 54a3dc0d, so that now there's a `POST /account/refresh` endpoint that refreshes the account data.

Thanks for the suggestions!